### PR TITLE
amp-selector: Support [disabled] binding

### PR DESCRIPTION
**Pull request by choumx**
Context: `b/115690618#comment5`

/to @jridgewell

### CODE REVIEW

File: extensions/amp-selector/0.1/amp-selector.js [hunk 1]
```diff
@@ -153,6 +148,16 @@ export class AmpSelector extends AMP.BaseElement {
     if (selected !== undefined) {
       this.selectedAttributeMutated_(selected);
     }
+    const disabled = mutations['disabled'];
+    if (disabled !== undefined) {
+      if (disabled) {
+        this.element.setAttribute('disabled', '');
```
Comment (jridgewell) [inline extensions/amp-selector/0.1/amp-selector.js]: Isn't the attribute already present?

Comment (choumx) [reply extensions/amp-selector/0.1/amp-selector.js]: Right-o.

File: extensions/amp-selector/0.1/amp-selector.js [hunk 2]
```diff
@@ -151,10 +151,8 @@ export class AmpSelector extends AMP.BaseElement {
     const disabled = mutations['disabled'];
     if (disabled !== undefined) {
       if (disabled) {
-        this.element.setAttribute('disabled', '');
         this.element.setAttribute('aria-disabled', 'true');
       } else {
-        this.element.removeAttribute('disabled');
```
Comment (jridgewell) [inline extensions/amp-selector/0.1/amp-selector.js]: Please confirm that amp-bind will drop this attribute.

**Comment by kul3r4** [comment]
@choumx has the validator been updated as well?

**Comment by choumx** [comment]
[disabled] has been valid for awhile now: https://github.com/ampproject/amphtml/blob/d3c7f705499f5e8d9a4471127bd2e620152f3ef3/extensions/amp-selector/validator-amp-selector.protoascii#L64

**Pull request merged**